pkg/controller/certificates/trigger/policies/gatherer.go

```diff
@@ -134,10 +134,10 @@ package policies
 //
 //
 // The third scenario C will help us understand the reason why
-// DataForCertificate fetch the "next" certificate request.
+// DataForCertificate has a "next" certificate request.
 //
 // DIAGRAM (C1): imagine that a user creates a certificate that contains a
-// mistake. The certificate will end up in failure mode and will be retried
+// mistake. The certificate will end up in failure state and will be retried
```

Comment:
I'm not sure if this change should be applied elsewhere in this PR 😉

Reply:
I think I'm the only person using the wording "mode" 😇

Reply:
I grepped the repo and could find one other occurrence that I wrote, I fixed it in 8f5a094b0